src/scales/scale.time.js

```diff
@@ -175,8 +175,20 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var majorUnitStart = startTick;
+			if (options.majorUnit) {
+				majorUnitStart = moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);
```

Comment:
Assignment `startTick` to `majorUnitStart` on line #179 is performed instead of `else` block of `if` statement at line #180
In other words we can write:
`
var majorUnitStart;
if (options.majorUnit) {
	majorUnitStart = moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);
} else {
	majorUnitStart = startTick;
}`